jetty-ant/src/main/java/org/eclipse/jetty/ant/JettyRunTask.java

```diff
@@ -147,13 +148,9 @@ public void setRequestLog(String className)
     {
         try
         {
-            this.requestLog = (RequestLog) Class.forName(className).newInstance();
+            this.requestLog = (RequestLog) Class.forName(className).getDeclaredConstructor().newInstance();
         }
-        catch (InstantiationException e)
-        {
-            throw new BuildException("Request logger instantiation exception: " + e);
-        }
-        catch (IllegalAccessException e)
+        catch (InstantiationException | NoSuchMethodException | InvocationTargetException | IllegalAccessException e)
```

Comment:
Convert this block to `catch (Exception e)` and put it after the `catch (ClassNotFoundException e)` below.